File: Java/백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int n = sc.nextInt();
        
        for (int i = 0; i < n; i++) {
            String cls = sc.next();
            
            if (cls.equals("Algorithm")) {
                System.out.println(204);
            } else if (cls.equals("DataAnalysis")) {
                System.out.println(207);
            } else if (cls.equals("ArtificialIntelligence")) {
                System.out.println(302);
            } else if (cls.equals("CyberSecurity")) {
                System.out.println("B101");
            } else if (cls.equals("Network")) {
                System.out.println(303);
            } else if (cls.equals("Startup")) {
                System.out.println(501);
            } else {
                System.out.println(105);
            }
        }
        
        sc.close();
    }
}

File: C++17/백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.cc
#include <iostream>
#include <string>
using namespace std;

int main(void) {
    int num;
    cin >> num;
    
    for (int i = 0; i < num; i++) {
        string cls;
        cin >> cls;
        
        if (cls == "Algorithm") cout << "204\n";
        else if (cls == "DataAnalysis") cout << "207\n";
        else if (cls == "ArtificialIntelligence") cout << "302\n";
        else if (cls == "CyberSecurity") cout << "B101\n";
        else if (cls == "Network") cout << "303\n";
        else if (cls == "Startup") cout << "501\n";
        else if (cls == "TestStrategy") cout << "105\n";
    }
    
    return 0;
}